styles/style.css

```diff
@@ -6,3 +6,183 @@ Green: #00B172
 White: #FFF
 
 */
+*{
+    font-family:Circular, Helvetica, Arial, sans-serif, IosFix ;
+    margin: 0px;
+
+}
+header{
+    
+    margin: 0;
+    position: fixed;
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+}
+.nav-bar{
+    position: fixed;
+    display: flex;
+    background-color: white;
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+}
+.nav-bar img{
+    max-width: 200px;
+    padding: 20px 30px;
+}
+.nav-bar ul{
+    display: flex;
+    justify-content: center;
+    width: 30%;
+    justify-content: space-evenly;
+    text-align: center;
+    margin-top: 40px;
+    margin-right: 40px;
+}
+.nav-bar li{
+    list-style-type: none;
+}
+.nav-bar a{
+    text-decoration: none;
+    color: black;
+}
+
+.section-1-div{
+    position: static;
+    margin-top: 0 ;
+}
+.section-1-div img{
+    width: 100%;
+    margin-top: 100px;
+    position: relative;
+    
+}
+#text-section-1{
+    font-size: 2rem;
+    color: white;
+    position: absolute;
+    top: 70%;
+    left: 50%;
+  transform: translate(-50%, -50%);
```

Comment:
What is the purpose of this key-value pair?

Reply:
It was actually a test, thought I deleted it , should have reviewed my code better sorry